Doc/library/random.rst

```diff
@@ -86,6 +86,11 @@ Bookkeeping functions
    .. versionchanged:: 3.2
       Moved to the version 2 scheme which uses all of the bits in a string seed.
 
+   .. deprecated:: 3.9
+      In the future, the *seed* must be one of the following types:
+      :class:`NoneType`, :class:`int`, :class:`float`, :class:`str`,
```

Comment:
There is no an entry for `NoneType`, so the link will not work.

Reply:
Is `float` because of `seed(time.time())`?

Reply:
Yes.